### DISCUSSION TITLE
Rooted firmware question

### DISCUSSION THREAD

**Discussion by ajiron** [Guilouz/Creality-Helper-Script-Wiki#438] [General]
Hi, I just got my K1 Max delivered yesterday, no previous experience with this printer model.  Is the rooted firmware available here need for the scripts to work?  If not, what are the advantages of having this firmware over the stock one of it can be rooted as well?
Thank you.

**Comment by Guilouz** [comment]
Firmware available on Wiki are official from Creality, you just need to enable root to use script.

**Reply by spitfire** [reply]
I was looking at the wiki (https://guilouz.github.io/Creality-K1-Series/firmwares/install-and-update-rooted-firmware/) and I have a related question: when I receive my K1 is it enough if I update it to the latest version (update directly from the pad) and then follow the "Enable Root access" section (before running your installation script) just like I did on my KE?

**Reply by ajiron** [reply]
I had issues with my USB drive so I didn't flash the one in this guide, I only updated to the latest OTA 1.3.3.5 and rooted from the menu.  I had no problem running the Helper scripts after that.

**Reply by Guilouz** [reply]
@spitfire Yes it's the same.

**Reply by imrutledge** [reply]
I upgraded my K1 max with the CFS. Before I updated, I reset to factory from the machine. Then I updated and tried to install the helper script. I get the "not found" error when I try to run the script.  Any idea how I can root? Thank you.

**Comment by fenix8k** [comment]
> Hi, I just got my K1 Max delivered yesterday, no previous experience with this printer model. Is the rooted firmware available here need for the scripts to work? If not, what are the advantages of having this firmware over the stock one of it can be rooted as well? Thank you.

Hi, you don't need a special "rooted" firmware, there is a option in the printer itself to root it, I don't remember where exactly but go through your printers config menus and you are going to find it
